pkg/devicetemplates/ttscsv.go

```diff
@@ -33,6 +36,8 @@ import (
 // TTSCSV is the device template converter ID.
 const TTSCSV = "the-things-stack-csv"
 
+var errGenerateSessionKeyID = errors.Define("generate_session_key_id", "failed to generate session key ID")
```

Comment:
Let's already give this code unavailable (`DefineUnavailable`), and drop the `failed to`.